src/nvim/plines.c

```diff
@@ -58,7 +58,7 @@ int win_get_fill(win_T *wp, linenr_T lnum)
 
   // be quick when there are no filler lines
   if (diffopt_filler()) {
-    int n = diff_check(wp, lnum);
+    int n = diff_check(wp, lnum, NULL);
```

Comment:
I would be tempted to redefine `diff_check` as.

```c
int diff_check(win_T *wp, linenr_T lnum)
{
  return diff_check_with_linestatus(wp, lnum, NULL);
}
```

Reply:
done